omnibus-arbitrum-sepolia.toml

```diff
@@ -0,0 +1,186 @@
+    "tomls/omnibus-arbitrum-sepolia/spot/usdc.toml",
+    "tomls/omnibus-arbitrum-sepolia/collaterals/usdc.toml",
+
+    # DAI
+    "tomls/oracles/pyth-dai.toml",
+    "tomls/omnibus-arbitrum-sepolia/spot/dai.toml",
+    "tomls/omnibus-arbitrum-sepolia/collaterals/dai.toml",
+]
+
+[setting.commitmentPriceDelay]
+defaultValue = "2"
+
+[setting.bigCapSettlementDelay]
+defaultValue = "2"
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.15"
+
+[setting.perps_keeper_cost_package]
+defaultValue = "arbitrum-gas-price-oracle:3.3.16"
+
+[setting.spot_market_package]
+defaultValue = "synthetix-spot-market:3.3.15"
+
+[setting.perps_market_package]
+defaultValue = "synthetix-perps-market:3.3.19"
+
+[setting.buyback_snx_package]
+defaultValue = "buyback-snx:3.3.15"
+description = "Do not update! This package isn't upgradeable. A version change will deploy a new contract."
+
+[setting.pyth_price_verification_address]
+defaultValue = "0x4374e5a8b9C22271E9EB878A2AA31DE97DF15DAF"
+description = "https://docs.pyth.network/documentation/pythnet-price-feeds/evm"
+
+[setting.pythFeedUrl]
+defaultValue = "https://api.synthetix.io/pyth-mainnet/api/get_vaa_ccip?data={data}"
+description = "Pyth settlement strategy config"
+
+[provision.pyth_erc7412_wrapper]
+source = "pyth-erc7412-wrapper:3.3.15"
+targetPreset = "<%= settings.target_preset %>"
+options.salt = "<%= settings.salt %>"
+options.pythAddress = "<%= settings.pyth_price_verification_address %>"
+
+[setting.owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.deployer]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.minimum_liquidity_ratio]
+defaultValue = "<%= parseEther('2.00') %>"
+description = "200% min market collateralization"
+
+[setting.account_timeout_withdraw]
+defaultValue = "<%= String(1 * 24 * 3600) %>"
+
+[setting.sc_pool_id]
+defaultValue = "1"
+
+[setting.pool_owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.settlementReward]
+defaultValue = "1"
+
+[setting.settlementMinimumUsdExchangeAmount]
+defaultValue = "<%= parseEther('0.000001') %>"
+
+[setting.settlementMaxRoundingLoss]
+defaultValue = "<%= parseEther('0.000001') %>"
+
+[setting.priceDeviationTolerance]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.spotStrictStalenessTolerance]
+defaultValue = "60"
+
+[setting.usdc_address]
+defaultValue = "0x75faf114eafb1BDbe2F0316DF893fd58CE46AA4d"
+description = "https://developers.circle.com/stablecoins/docs/usdc-on-test-networks"
+
+[setting.pythUsdcFeedId]
+defaultValue = "0xeaa020c61cc479712813461ce153894a96a6c00b21ed0cfc2798d1f9a9e9c94a"
+description = "https://pyth.network/developers/price-feed-ids#pyth-evm-mainnet"
+
+[setting.synthUsdcMaxMarketCollateral]
+defaultValue = "<%= parseEther(String(500_000)) %>"
+
+[setting.synthUsdcAsyncFixedFee]
+defaultValue = "<%= parseEther('0.1') %>" # FIXME
+
+[setting.synthUsdcSkewScale]
+defaultValue = "<%= parseEther(String(100_000)) %>" # FIXME
```

Comment:
Skew scale/fee shouldn't be necessary here

Reply:
What value "disables" skew scale exactly?

Reply:
Just not setting it should be fine, effectively leaving the scale at zero. See https://github.com/Synthetixio/synthetix-v3/tree/main/markets/spot-market#auto-rebalancing-skew-fee

Reply:
removed skew configuration step